test/acceptance/public-api-test.js

```diff
@@ -1475,5 +1474,35 @@ describe('public api', function () {
       expect(result.output).toEqual('\uFEFF<button type="button">LOL, Click me!</button>');
       expect(result.isFixed).toEqual(true);
     });
+
+    it("[.html] files with comments won't cause errors", async function () {
```

Comment:
Can you tweak this to mention DOCTYPE? The issue isn't really with comments, its due to doctype...